src/qml/qmlplayermanagerproxy.cpp

```diff
@@ -60,7 +60,7 @@ void QmlPlayerManagerProxy::loadLocationToPlayer(
 }
 
 // static
-QmlPlayerManagerProxy* QmlPlayerManagerProxy::create(QQmlEngine* pQmlEngine, QJSEngine* pJsEngine) {
+QmlPlayerManagerProxy* QmlPlayerManagerProxy::create(QJSEngine* pJsEngine, QObject* parent) {
```

Comment:
Changing the order of arguments here to closer match the signature of `QmlPlayerManagerProxy`.

Reply:
Also note that the ordering was likely chosen deliberately because it matches the Qt example here: https://doc.qt.io/qt-6/qqmlengine.html#QML_SINGLETON

Reply:
Hmm… Wait, it's supposed to be a singleton instance but it doesn't implement the singleton pattern. The `create` static method doesn't check the existance of a static field like it's done on https://doc.qt.io/qt-6/qqmlengine.html#QML_SINGLETON.